From e57fd5f1da0e5efb968ef72e0b80e15ebe0dce8e Mon Sep 17 00:00:00 2001
From: Mamoru TASAKA <mtasaka@fedoraproject.org>
Date: Wed, 28 May 2014 16:08:15 +0900
Subject: [PATCH] test-passwd: add skeleten definition for reset_watchdog_timer

Now lock.c needs reset_watchdog_timer definition. for test-passwd,
just use skeleton definition
---
 driver/test-passwd.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/driver/test-passwd.c b/driver/test-passwd.c
index 66cd3d4..0745274 100644
--- a/driver/test-passwd.c
+++ b/driver/test-passwd.c
@@ -80,6 +80,8 @@ idle_timer (XtPointer closure, XtIntervalId *id)
   XPutBackEvent (si->dpy, &fake_event);
 }
 
+/* Add fake function */
+void reset_watchdog_timer(saver_info *si, Bool on_p){}
 
 static int
 text_auth_conv (
-- 
1.9.3

